api/src/main/java/jakarta/batch/runtime/BatchRuntime.java

```diff
@@ -43,28 +43,33 @@
 	* @return JobOperator instance.
 	*/
 	public static JobOperator getJobOperator() {
+		JobOperator operator = null;
+		if (System.getSecurityManager() == null) {
+			for (JobOperator op : ServiceLoader.load(JobOperator.class)) {
+				operator = op;
+				break;
+			}
```

Comment:
Just looking at the else path...  I don't see that we need the null check.  Can we add back the logging from there?

Reply:
added the fine log; and removed the redundant null check in else path.